## 4.	AT&T was running commercials in 1990 aimed at luring back customers who had switched to one of the other long-distance phone service providers. One such commercial shows a businessman trying to reach Phoenix and mistakenly getting Fiji, where a half-naked native on a beach responds incomprehensibly in Polynesian. When asked about this advertisement, AT&T admitted that the portrayed incident did not actually take place but added that this was an enactment of something that “could happen.” Suppose that one in 200 long-distance telephone calls is misdirected. What is the probability that at least one in five attempted telephone calls reaches the wrong number? (Assume independence of attempts.

In [1]:
from scipy.stats import binom

### What is the probability that at least one in five attempted telephone calls reaches the wrong number? (Assume independence of attempts.)

In [5]:
n=5
# success is telephone calls reaches the wrong number.
# p= 1/200
p=0.005

In [6]:
k=0
1- binom.pmf(n=n, p=p,k=k)

0.02475124687812502

## the probability that at least one in five attempted telephone calls reaches the wrong number is 0.02475